Sources/PrimerSDK/Classes/Extensions & Utilities/PrimerButton.swift

```diff
@@ -9,7 +9,15 @@
 
 import UIKit
 
+///
+/// Reserve the name for all primer buttons. If you need to extend UIButton, extend and use this one instead, so we
+/// don't expose unnecessary functionality.
+///
```

Comment:
will this be the renaming pattern going forward? I.e. when we want to make a component deprecated